packages/discovery-react-components/src/components/StructuredQuery/messages.ts

```diff
@@ -87,5 +91,6 @@ export const defaultMessages: Messages = {
   valueInputPlaceholderText: 'Enter value',
   addRuleRowText: 'Add rule',
   removeRuleRowButtonIconDescription: 'Remove rule',
-  addRuleGroupText: 'Add group of rules'
+  addRuleGroupText: 'Add group of rules',
+  fieldDropdownLoadingText: 'Fetching project fields'
```

Comment:
`Fetching` seems more technical and less friendly than `Loading` to me for the default text here.

Reply:
nitpick - can we put this next to the other `field*` messages above?